graph_embeddings.py

```diff
@@ -15,9 +15,7 @@ def __init__(self, triples_file,output_dir,input_dir,embedding_dimensions):
     def check_file_existence(self,embeddings_file):
         exists = 'false'
         for fname in os.listdir(self.output_dir):
-           # print(fname)
-            if bool(re.search("Embeddings", fname)):
-               # print('match')
+            if bool(re.search(embeddings_file, fname)):
```

Comment:
where is embeddings_file referenced?

Reply:
it's used in line 96 to read in the embeddings object as emb

Reply:
yep, I didn't expand the view enough. my bad.